### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by bac**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3950/
Test PASSed.

**Comment by makyo** [comment]
:+1: Thanks for this!

File: HACKING.rst [hunk 1]
```diff
@@ -5,77 +5,120 @@ HACKING
...
+   rm $CHARM/releases/*.bz2
+   cp $GUI/dist/*.bz2 $CHARM/releases/
 
-For production:
+3. Bootstrap the environment and deploy the charm:
 
 ::
 
-   make gui-prod
+   juju switch <your preferred provider>
```
Comment (urosj) [inline HACKING.rst]: s/provider/model

**Comment by urosj** [comment]
:+1:

**Comment by urosj** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1358

**Comment by urosj** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**